src/plugins/home/public/application/components/home_app.js

```diff
@@ -35,6 +35,7 @@ import { Home } from './home';
 import { FeatureDirectory } from './feature_directory';
 import { TutorialDirectory } from './tutorial_directory';
 import { Tutorial } from './tutorial/tutorial';
+import { OllyHomepage, PlaygroundHomepage } from './homepage';
```

Comment:
for the sake of time, ok but it should be generic in the sense that its like "new homepage", "feature homepage", "default homepage"